core/tests/protractor_desktop/navigation.js

```diff
@@ -213,7 +213,12 @@ describe('Static Pages Tour', function() {
     expect(element(
       by.css('.protractor-test-thanks-page')).isPresent()).toBe(true);
   });
-
+  it('visits the Welcome Guide Page', function() {
+    browser.get('/welcomeguidepage');
```

Comment:
Thanks for checking that we got to the right page using an element instead of the URL! Can you also navigate to the page the same way a user would? For example, clicking on a link/button that leads to this page

Reply:
Yes, you can! You can click on the dropdown menu on the about page and click on "welcome guide" or you can use the sidebar.

Reply:
Hi @sm6957 sorry, I realize my comment was ambiguous. I was asking that you (I'm assuming you were working with @kmaraj to create this PR) please change the test to access the welcome guide page just like a user would instead of going directly to the URL. This way you also test the navigation flow. As a general principle, we try to write end-to-end tests to follow the same steps a user would.

Reply:
Thanks for rasing this comment @U8NWXD. I have one more question: This page doesn't show "Loading..." message then do we need `waitFor.pageToFullLoad` here? If no, then how we will confirm that the page has fully loaded before testing the next line?

Reply:
@DubeySandeep @kmaraj `waitFor.pageToFullyLoad` waits for the loading message to disappear, so its a bit of a misnomer. If no loading message is shown, then we have to wait for some other element on the page. I usually watch the page load, possibly over a throttled connection to slow down loading, to see what loads last.

Reply:
test has been added!